portfolio/src/main/webapp/script.js

```diff
@@ -97,25 +98,25 @@ function createListElement(key, text) {
 
 // Check if the user is logged in or not, only display the comment function
 // if they are logged in.
-var xhttp = new XMLHttpRequest();
+const xhttp = new XMLHttpRequest();
 xhttp.onreadystatechange = function() {
   if (this.readyState == 4 && this.status == 200) {
     commentTitle = document.getElementById('comment-header');
     if (xhttp.responseText.includes('<p>You are logged in!</p>')) {
-      var form = document.createElement('form');
+      const form = document.createElement('form');
       form.setAttribute('method', 'POST');
       form.setAttribute('action', '/data');
 
-      var text = document.createElement('p');
+      const text = document.createElement('p');
```

Comment:
feedback_prompt?

Reply:
renamed to feedbackPrompt to be more descriptive